src/components/Navbar.jsx

```diff
@@ -13,6 +13,41 @@ const Navbar = () => {
   const [mobileOpen, setMobileOpen] = useState(false)
   const { theme, toggleTheme } = useContext(ThemeContext)
   const { navbar } = theme
+  const navLinks = [
+    {
+      name: "Home",
+      path: "/",
+    },
+    {
+      name: "Projects",
+      path: "/projectspage",
+    },
+    {
+      name: "Docs",
+      path: "/docs",
+    },
+    {
+      name: "Contributors",
+      path: "/contributorspage",
+    },
+  ]
+
+  const navLinkEls = navLinks.map((navLink) => (
+    <li key={navLink.path}>
+      <NavLink
+        to={navLink.path}
+        className={({ isActive }) =>
+          `inline-block py-2 px-3 text-center font-bold  rounded-md ${
+            theme.mode === "light"
```

Comment:
make theme mode by default `dark` 😢 dark theme helps is ❤️ 
```suggestion
            theme.mode === "dark"
```